basics.tex

```diff
@@ -109,10 +109,12 @@ \section{Basic Data Types}\label{s:basics-data-types}
 the type of null is object
 \end{minted}
 
-As the example above shows,
-we use \texttt{let} to define a \gref{g:variable}{variable},
-put values separated by commas inside \texttt{{[}{]}} to create an \gref{g:array}{array},
-and use \texttt{for...of} to loop over the values in that array.
+As the example above shows, we create an array of values to be looped through called 
```

Comment:
1. Please use semantic line breaks (i.e., break the line on each phrase or sentence) - it makes it a lot easier to move sentence fragments around, and and reduces the risk of of having duplicated or missing words.
2.  "to loop through" rather than "to be looped through"